### PR TITLE
Mpool documentation and config cli command

### PR DESCRIPTION
**Pull request by vyzo**
- Adds a cli command to get/set the mpool config
- Adds some documentation for the message pool. It's not yusef caliber, but it's a start.

### CODE REVIEW

File: documentation/en/mpool.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+```
+
+The meaning of these fields is as follows:
+- `PriorityAddrs` -- these are the addresses of actors whose pending messages should always
+  be included in a block during message selection, regardless of profitability.
+  Miners should configure their own worker addresses so that they include their own messages
+  when they produce a new block.
+- `SizeLimitHigh` -- this is the maximum number of pending messages before triggering a
+  prune in the message pool. Note that messages from priority addresses are never pruned.
+- `SizeLimitLow` -- this is the number of pending messages that should be kept after a prune.
```
Comment (magik6k) [inline documentation/en/mpool.md:126]: Would be nice to note defaults

File: documentation/en/mpool.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+for inclusion to the next block in order to maximize its reward. The
+problem however is NP-hard (it's an instance of knapsack packing),
+further confounded by the fact that miners don't communicate their
+tickets to each other. So at best we can approximate the optimal
+selection in a reasonable amount of time.
+
+The mpool employs a sophisticated algorithm for selecting messages for
+inclusion, given the ticket quality of a miner. The ticket quality
+reflects the probability that a block will execute in the
+tipset. Given the ticket quality the algorithm computes the
```
Comment (magik6k) [inline documentation/en/mpool.md:146]: The wording is mildly confusing here. All blocks in a tipset execute, it's just the first one to include a given message which gets the reward

Comment (vyzo) [reply documentation/en/mpool.md:146]: Yeah, the word should probably be message.

File: documentation/en/mpool.md (hunk 1 above)
Comment (vyzo) [reply documentation/en/mpool.md:126]: Sure, will do!

**Pull request merged**